### ISSUE TITLE
The application does not start 

### ISSUE THREAD

**Issue by nuphion** [jarrettgilliam/AutoGame#13]
Windows 10 x64

Installs fine but application does not open. No errors and nothing in task manager. 

Let me know what you need to know to debug/diagnose.

**Comment by jarrettgilliam** [comment]
Hey Rob! A couple of things we could check here. What happens if you start the "C:\Program Files\AutoGame\AutoGame.exe" executable from within Command Prompt? Also, are there any errors in the Application log in Event Viewer?

<img width="200" alt="image" src="https://user-images.githubusercontent.com/5099690/183221207-4d61bd71-aa87-4e85-9d3f-9e227e622316.png">

**Comment by nuphion** [comment]
There were logs in Event Viewer but not useful ones.

I did figure out the problem eventually.

I already had an early version of .NET 6 installed from last year, after trying a few things that didn't work I updated .NET and AutoGame started working. So I guess either my previous installation of .NET was too old or perhaps broken somehow.

I've built the solution locally as a self contained application and this is the version of AutoGame I'm using now, to avoid relying on the globally installed runtime. Maybe that's a potential improvement for you to consider but your choice.

Thanks for making this, it should come in handy.
